ignite/cmd/scaffold_type.go

```diff
@@ -10,7 +10,7 @@ import (
 func NewScaffoldType() *cobra.Command {
 	c := &cobra.Command{
 		Use:     "type NAME [field]...",
-		Short:   "Scaffold only a type definition",
+		Short:   "Type definition",
```

Comment:
I'm not sure I like the verb removal from the short description. What's the rationale behind?

Reply:
When verbs are different (like with `ignite --help`), I agree! When we're already looking up the contents of a "verb" (`ignite scaffold --help`), I think repetition makes it harder to read.

```
Available Commands:
  chain       New Cosmos SDK blockchain
  module      Custom Cosmos SDK module
  list        CRUD for data stored as an array
  map         CRUD for data stored as key-value pairs
  single      CRUD for data stored in a single location
  type        Type definition
  message     Message to perform state transition on the blockchain
  query       Query for fetching data from a blockchain
  packet      Message for sending an IBC packet
  vue         Vue 3 web app template
  react       React web app template
```
vs
```
Available Commands:
  chain       Scaffold a new Cosmos SDK blockchain
  module      Scaffold a custom Cosmos SDK module
  list        Scaffold CRUD for data stored as an array
  map         Scaffold CRUD for data stored as key-value pairs
  single      Scaffold CRUD for data stored in a single location
  type        Scaffold a type definition
  message     Scaffold a message to perform state transition on the blockchain
  query       Scaffold a query for fetching data from a blockchain
  packet      Scaffold a message for sending an IBC packet
  vue         Scaffold a Vue 3 web app template
  react       Scaffold a React web app template
```

Reply:
Oh nice indeed <3